Firestore/core/src/model/mutable_document.cc

```diff
@@ -87,18 +83,27 @@ MutableDocument& MutableDocument::SetHasLocalMutations() {
   return *this;
 }
 
+size_t MutableDocument::Hash() const {
+  return key_.Hash();
+}
+
 std::string MutableDocument::ToString() const {
   std::stringstream stream;
   stream << "MutableDocument(key=" << key_ << ", type=" << document_type_
-         << ", version=" << version_ << ", value=" << value_
+         << ", version=" << version_ << ", value=" << CanonicalId(value_->Get())
```

Comment:
Question: is it necessary to call `CanonicalId` and `Get` rather than just do `<< *value_`?